api/src/index.js

```diff
@@ -11,21 +11,97 @@ app.use(bodyParser.json());
 
 const apiRouter = express.Router();
 
-// You can delete this route once you add your own routes
-apiRouter.get("/", async (req, res) => {
-  const SHOW_TABLES_QUERY =
-    process.env.DB_CLIENT === "pg"
-      ? "SELECT * FROM pg_catalog.pg_tables;"
-      : "SHOW TABLES;";
-  const tables = await knex.raw(SHOW_TABLES_QUERY);
-  res.json({ tables });
-});
+//future-meals
+apiRouter.get("/future-meals", async (req, res) => {
+  try {
+    const futureMeals = await knex.raw("Select * From Meal WHERE Meal.when > NOW()")
+    if (futureMeals[0].length === 0) {
+    return res.status(404).json("There are no meals for the future.");
+    }
```

Comment:
In such cases you should not return a `Not Found` error. If there are no meals, and the user wants to see all of the future ones, then returning an empty array is a valid answer.